packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -43,7 +43,9 @@ async function compileFixture({ input, dirname }: { input: string; dirname: stri
             }),
         ],
         onwarn(warning) {
-            warnings.push(warning);
+            if (warning.message.includes('LWC1187')) {
```

Comment:
Adding a comment to explain why we're only looking for 1187 would be helpful.

Reply:
Added in e0e56d3aed29325824c6b84872e5905a60210a25!